app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,278 @@
+<p>
+    Appwrite provides authentication for many different use cases to fit the needs of developers. 
+    Appwrite manages authentication with a combination of users and sessions. 
+    Accounts can be created in many different ways, such as through an anonymous session, email and password, OAuth authentication, magic URLs, and more.
+</p>
+
+<h2><a href="/docs/authentication#account-vs-user-service" id="account-vs-user-service">Account vs User Service</a></h2>
+
+<p>
+    While the Users API is integrated from the server-side and operates in an admin scope with access to all your project users, the Account API operates in the scope of the currently logged-in user and usually uses a client-side integration.
+</p>
+
+<p> 
+    Some of the Account API methods are available from the server SDK when you authenticate with JWT. This allows you to perform server-side actions on behalf of your project user.
+</p>
+
+<h2><a href="/docs/authentication#create-an-account" id="create-an-account">Create An Account</a></h2>
+
+<p>
+    A user account in Appwrite is the primary way to access information for a given project. Accounts can be created in many different ways, including email/password, anonymously, using OAuth providers, phone authentication, and more. Users can create and manage authentication sessions through the REST API or client SDKs.
```

Comment:
>Users can create

I think you mean an Appwrite user. We should call those who develop with Appwrite as developers or Appwrite developers.

This way it's clear they're not a "user" that's created through the account service.